src/test/java/com/rocketden/main/ProblemTests.java

```diff
@@ -51,6 +53,26 @@
   @LocalServerPort
   private int port;
 
+  /**
+   * Sets up necessary test fields and fixtures.
+   * (Called once before the rest of the test case methods.)
+   */
+  public ProblemTests() {
+    // Create "Sort Array" problem JSON string.
+    sortArrayProblem = new ProblemDto();
+    sortArrayProblem.setName("Sort an Array");
+    sortArrayProblem.setDescription
+      ("Sort an array from lowest to highest value.");
+    sortArrayProblemJson = Utility.convertObjectToJsonString(sortArrayProblem);
+
+    // Create "Find Maximum" problem JSON string.
+    findMaxProblem = new ProblemDto();
+    findMaxProblem.setName("Find Maximum");
+    findMaxProblem.setDescription
+      ("Find the maximum value in an array.");
```

Comment:
Is there a reason why the parentheses are on a new line, like to do with max line length? If so, I get that, though imo it does look a bit funky.

Reply:
Yes, it's for max line length (I try to keep it less than 80 characters). I agree it looks funky, it doesn't bother me too much but if you have a fix / suggestion we can change this.